recipes/sampledb/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sampledb" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "95755272543b8cc2852ca73731b669e67ac9f8075d008e6c7c05fad2dff679fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [not py36]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - xlrd
+    - xlsxwriter
+    - pymongo
+    - pandas
+    - jsonschema
+    - xonsh
+    # Note that the qr writer requires latex and I don't want to get into
+    # cf tex distributions
+
+test:
+  imports:
+    - sampledb
+    - sampledb.databasesearcher
+    - sampledb.datapublisher
+
+about:
+  home: https://github.com/diffpy/sampledb
+  license: BSD-3-clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A DB for sample and synthesis metadata information'
+
+  doc_url: http://www.diffpy.org/sampledb/
+  dev_url: https://github.com/diffpy/sampledb
+
+extra:
+  recipe-maintainers:
+    - CJ-Wright
+    - justcalamari
```

Comment:
@justcalamari -- are you ok being added as a co-maintainer of the `sampledb` on conda-forge?

Reply:
Yes, I am okay with that.